config/jobs/kubernetes/wg-k8s-infra/canaries/sig-node-containerd.yaml

```
@@ -275,6 +303,43 @@ periodics:
   annotations:
     testgrid-dashboards: wg-k8s-infra-canaries
     testgrid-tab-name: containerd-node-e2e-1.4-canary
+- name: ci-containerd-node-e2e-1-5-canary
+  interval: 1h
+  cluster: k8s-infra-prow-build
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210721-2b77449-master
+      args:
+      - --root=/go/src
+      - --repo=k8s.io/kubernetes=release-1.21
```

why testing with 1.21?
```suggestion
      - --repo=k8s.io/kubernetes=release-1.22
      - --repo=github.com/containerd/containerd=release/1.5
```